markers/polarion.py

```diff
@@ -13,3 +14,10 @@ def extract_polarion_ids(item):
         return None
 
     return map(str, polarion.args)
+
+
+@pytest.fixture(autouse=True)
+def _update_polarion_in_junit(request, record_xml_property):
+    marker = request.node.get_marker("polarion")
```

Comment:
**REQUIRED** please use extract_polarion_ids 
**OPTIONAL** please investigate the effect of recording more than one polarion test id since i believe some tests are marked with more than one polarion id

Reply:
So I changed my code to work with multi polarion marks

```python
@pytest.fixture(autouse=True)
def _update_polarion_in_junit(request, record_xml_property):
    """Adds the supplied test case id to the xunit file as a property"""
    from functools import partial
    record_id_to_xml = partial(record_xml_property,("test_id",))
    ids = extract_polarion_ids(request.node)
    if ids is not None:
        map(record_id_to_xml, ids)
```
This is an example for a test case with multiple polarion markes

```xml
  <testcase classname="cfme.tests.containers.fake_test" file="cfme/tests/containers/fake_test.py" line="14" name="test_nothing2[4]" time="38.2410292625">
    <properties>
      <property name="('test_id',)" value="CMP-004"/>
      <property name="('test_id',)" value="CMP-005"/>
    </properties>
  </testcase>
```